src/app/dashboard/emsZones/objects/connections.tsx

```
@@ -0,0 +1,1003 @@
+						makeLineSegment(
+							points,
+							conn,
+							conn.status1 === "okay" ? "#00ff00" : "#ff0000"
+						),
+
+						// eslint-disable-next-line react/jsx-key
+						<polygon
+							fill={conn.status1 === "okay" ? "#00ff00" : "#ff0000"}
+							points={`${Ax1 - 10},${Ay1 - 10} ${Ax1 + 10},${Ay1} ${Ax1 - 10},${Ay1 + 10}`}
+							transform={`rotate(${angleA}, ${Ax1}, ${Ay1})`}
+						/>,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)